config.go

```diff
@@ -39,14 +39,14 @@ const (
 // Config represents the configuration for the command.
 type Config struct {
 	DataDir string `toml:"data-dir"`
-	Host    string `toml:"host"`
+	Host    string `toml:"bind"`
```

Comment:
I saw `Host` is used by `pilosa/tools/creator`, is changing it break tools? I'm just not sure if any other app calls Config.Host directly might be broken from the changing struct's field